frontend/src/features/properties/selector/PropertySelectorSubForm.tsx

```diff
@@ -33,6 +37,12 @@ export const PropertySelectorSubForm: React.FunctionComponent<IPropertySelectorS
         <SectionField label="PID">{pid}</SectionField>
         <SectionField label="Plan #">{planNumber}</SectionField>
         <SectionField label="Address">{address}</SectionField>
+        <SectionField label="Region">
+          {[region, regionName].filter(Boolean).join(' - ')}
+        </SectionField>
+        <SectionField label="District">
+          {[district, districtName].filter(Boolean).join(' - ')}
+        </SectionField>
```

Comment:
Took the liberty to include the region and district in the property picker sub-form as I think it is quite useful information. Happy to take it out or modify if UX deems it not useful. See screenshot for new look